api/nodecontrol/client_test.go

```diff
@@ -0,0 +1,943 @@
+package nodecontrol
```

Comment:
I suggest adding a file or internal package that defines a set of standard assertion functions like [nats-server](https://github.com/nats-io/nats-server/blob/62de2cd3287f0ae21255f08c92efe379b318fa42/server/test_test.go#L58-L169). It shortens the tests a bit and keeps the output messages for simple asserts consistent.

Reply:
[126eead](https://github.com/synadia-io/nex/pull/473/commits/126eeadbf2414db6e2baa1c6eabd7f6c46bb250c) brings in [github.com/carlmjohnson/be](http://github.com/carlmjohnson/be)